src/v/security/audit/schemas/application_activity.h

```diff
@@ -89,6 +89,13 @@ class api_activity final : public ocsf_base_event<api_activity> {
     status_id _status_id;
     api_activity_unmapped _unmapped;
 
+    ss::sstring api_info() const final {
+        if (_http_request) {
```

Comment:
nit: I _think_ our style is to use `.has_value()` rather than relying on the bool operator, but I could be mistaken